### PR TITLE
Added Nexus doc and endpoints description

### PR DESCRIPTION
**Pull request by talha0324**

### CODE REVIEW

**Comment by stakater-user** [comment]
@talha0324 Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-132-2dc589ba`

**Comment by stakater-user** [comment]
@talha0324 Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-132-fb173e6a`

File: content/sre/repository/nexus.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Nexus
+
+## Introduction
+
+[Nexus](https://www.sonatype.com/products/repository-pro) is a repository manager that can store and manage components, build artifacts, and release candidates in one central location. It can host multiple type of repositories like docker, helm, maven and more.
+
+## Customer nexus endpoints and their purpose
```
Comment (rasheedamir) [inline content/sre/repository/nexus.md:7]: @talha0324 we should bit more details

- that we configure these separate endpoints in SAAP for this reason
- also mention that we support whitelisting
- convert this to table instead I will suggest

**Comment by stakater-user** [comment]
@talha0324 Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-132-d155b3cd`

**Comment by stakater-user** [comment]
@talha0324 Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-132-1694c275`

**Comment by stakater-user** [comment]
@talha0324 Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-132-c40cbf79`

**Comment by stakater-user** [comment]
@talha0324 Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-132-f1e75eef`

**Review by rasheedamir — APPROVED** [review]

**Comment by stakater-user** [comment]
@rasheedamir Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:v0.0.79`

**Pull request merged**